### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by TheDementedSalad**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x ] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x ] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x ] The Auto Splitter has an open source license that allows anyone to fork and host it.

I have permission from Meta, since the autosplitter has been completely made now we agreed it would be better for me to take over so I can update it when needs be

### CODE REVIEW

**Comment by just-ero** [comment]
@LiterallyMetaphorical Are you alright with these changes?

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
+         <AutoSplitter>
+       <Games>
+           <Game>Exodus from the Earth</Game>
+       </Games>
+       <URLs>
+           <URL>https://raw.githubusercontent.com/TheDementedSalad/Exodus-From-The-Earth-Autosplitter/main/Exodus%20From%20The%20Earth.asl</URL>
+       </URLs>
+       <Type>Script</Type>
+       <Description>Auto Splitting and Load Removal are available. (By TheDementedSalad &amp; Ellie &amp; SabulineHorizon)</Description>
+     </AutoSplitter>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:32]:
```suggestion
    <AutoSplitter>
        <Games>
            <Game>Exodus from the Earth</Game>
        </Games>
        <URLs>
            <URL>https://raw.githubusercontent.com/TheDementedSalad/Exodus-From-The-Earth-Autosplitter/main/Exodus%20From%20The%20Earth.asl</URL>
        </URLs>
        <Type>Script</Type>
        <Description>Auto Splitting and Load Removal are available. (By TheDementedSalad &amp; Ellie &amp; SabulineHorizon)</Description>
    </AutoSplitter>
```

**Comment by LiterallyMetaphorical** [comment]
Yep all good!

**Comment by TheDementedSalad** [comment]
Any chance this can me merged when possible? It's kinda needed for the speedrun competition going on atm

**Comment by wooferzfg** [comment]
Sorry for the delay!

**Comment by just-ero** [comment]
> Any chance this can me merged when possible? It's kinda needed for the speedrun competition going on atm

It would be nice if you could actually address the requested changes next time (https://github.com/LiveSplit/LiveSplit.AutoSplitters/pull/1175#pullrequestreview-1284042739). This would have been merged hours ago had you done that.

**Comment by TheDementedSalad** [comment]
> > Any chance this can me merged when possible? It's kinda needed for the speedrun competition going on atm
> 
> It would be nice if you could actually address the requested changes next time ([#1175 (review)](https://github.com/LiveSplit/LiveSplit.AutoSplitters/pull/1175#pullrequestreview-1284042739)). This would have been merged hours ago had you done that.

I was on mobile and had only seen you ask if Meta gave me permission, apologies

**Comment by wooferzfg** [comment]
Feel free to open a separate PR to address the whitespace issues. (I didn't see it as a blocker to merging)

**Pull request merged**